GetStream/stream-chat-android
# [3919] App crashes when reacting to a message in a thread

### 🎯 Goal

Avoid crashes while reacting to a message in a thread.

### 🛠 Implementation details

Store replies coming `messages/{parent_id}/replies` endpoint

### 🧪 Testing

- install a fresh build
- log in using any user who has channels with threads
- open a channel with treads
- open any thread
- leave a reaction on other's user reply

**Actual**: Observe a crash
**Expected**: Reaction is made with no issues


### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_
